aodncore/util/external/awsretry.py

```diff
@@ -0,0 +1,136 @@
+#!/usr/bin/env python
+
+from __future__ import absolute_import, division, print_function
+from __future__ import unicode_literals
+
+import logging
+import re
+import time
+from functools import wraps
+
+import boto
+import botocore
+
+__author__ = 'Allen Sanabria'
+__version__ = '1.0.2'
```

Comment:
Should there be a link to the source of this code here?

Reply:
It's already here: https://github.com/aodn/python-aodncore/pull/61/files#diff-c9d6c5c00c596b6bbd4fbc00452260aaR5

I didn't want to modify the source unnecessarily, so we could (in theory, but unlikely) upgrade it in place. It is MIT licensed, so GPLv3 compatible, but if we really had to replace it, not a big deal either.

Reply:
:+1: Ok, that makes sense.

Reply:
Want to make sure we're good open source citizens ;) Also, have fixed the merge conflict, which was caused by https://github.com/aodn/python-aodncore/pull/57